test/integ_tests/requirements.txt

```diff
@@ -1,13 +1,31 @@
-amazon-braket-sdk
```

Comment:
These requirements are left unpinned to allow to build with the latest versions. We should not make the two files equivalent.